### PR TITLE
fix copyright for nuget compliance

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <!-- This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion -->
     <Version>$(MsalClientSemVer)</Version>
 
-    <Copyright>Copyright (c) Microsoft Corporation. All rights reserved.</Copyright>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
```
Comment (henrik-me) [inline msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj:20]: >© [](start = 15, length = 1)

Is the recommended practice?

**Review by henrik-me — APPROVED** [review]
:shipit:

File: msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj (hunk 1 above)
Comment (jennyf19) [reply msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj:20]: image

Comment (jmprieur) [reply msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj:20]: the (c) is not equivalent to "©"  ?

Comment (jennyf19) [reply msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj:20]: @jmprieur not according to the email we got. I just forwarded it to the team.

**Pull request merged**